google-cloud-bigquery/acceptance/bigquery/dataset_reference_test.rb

```diff
@@ -108,11 +108,12 @@
   end
 
   it "deletes itself and knows it no longer exists" do
-    _(dataset.exists?).must_equal true
-    dataset.tables.all(&:delete)
-    _(dataset.delete).must_equal true
-    _(dataset.exists?).must_equal false
-    _(dataset.exists?(force: true)).must_equal false
+    dataset_2_id = "#{prefix}_dataset_delete"
```

Comment:
Unclear to me how these changes relates to the rest, just adding an additional testing dataset?  Same with the acceptance test for table reference.

Reply:
Yes, just some maintenance work on flakey tests unrelated to the feature.